FinniversKit/Sources/Base Components/ButtonStyle.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+public struct InlineFlatStyle: ButtonStyle {
+    let font: Font
```

Comment:
Minor thing, but I think properties here and in the other structs below could be private.